charactersheet/charactersheet/models/common/item.js

```diff
@@ -43,7 +43,7 @@ function Item() {
     });
 
     self.itemWeightLabel = ko.pureComputed(function() {
-        return self.itemWeight() ? self.itemWeight() + ' lbs.' : '';
+        return self.itemWeight() >= 0 ? self.itemWeight() + ' lbs.' : '';
```

Comment:
We need to for value and if it's greater than zero. If not, we get this:
![image](https://cloud.githubusercontent.com/assets/7286387/22857357/90da1526-f058-11e6-86cf-86e727039fc4.png)